src/pages/guides/using/aem/aem_keystore_setup.md

```diff
@@ -0,0 +1,81 @@
+---
```

Comment:
why adding this here, it's already in the deprecated folder... right ?

Reply:
Removed this as well.